pipeline/run.sh

```diff
@@ -307,10 +307,12 @@ col_id=\$(sbatch --job-name=5-collectSamples_${scanmode}_${name} --time=${job_5_
 # hmdb_part.R
 echo "#!/bin/sh
 
-/hpc/local/CentOS7/common/lang/R/3.2.2/bin/Rscript ${scripts}/hmdb_part.R ${outdir} ${scanmode} ${db} ${ppm}
+/hpc/local/CentOS7/common/lang/R/3.2.2/bin/Rscript ${scripts}/hmdb_part.R ${outdir} ${scanmode} ${db} ${ppm} ${stdrun} ${stdrun_loc}
 " > ${outdir}/jobs/hmdb_part/${scanmode}.sh
 hmdb_id_1=\$(sbatch --job-name=hmdb_part_${scanmode}_${name} --time=${job_hmdb1_time} --mem=${job_hmdb1_mem} --output=${outdir}/logs/hmdb_part/${scanmode}.o --error=${outdir}/logs/hmdb_part/${scanmode}.e ${global_sbatch_parameters} ${outdir}/jobs/hmdb_part/${scanmode}.sh)
 echo "${hmdb_id_1}" > ${outdir}/logs/hmdb_1
+echo "${stdrun}"  > ${outdir}/logs/stdrun.log
+echo "${stdrun_loc}"  > ${outdir}/logs/stdrun_loc.log
```

Comment:
- Why do you need two separate log files? Seems a bit much for just logging yes/no and a location.
- 'stdrun' = standard run? So two words -> `std_run` or just use `standard_run`.
- `stdrun_loc` = a dir/folder location, other parameters are called `..._dir`. Furthermore this seems to be a location of hdmb parts so -> `hmdb_parts_dir`.

Reply:
These 2 log files were used in testing, removed both for production.
stdrun replaced with standard_run
stdrun_loc replaced with hmdb_parts_dir